mainapp/templates/nav.html

```diff
@@ -62,11 +62,18 @@
                         <a class="nav-link" href="{% url 'favlist' %}">Favorites</a>
                         {% endif %}
                     </li>
-                    <li class="nav-item active">
+                    <li class="nav-item active ">
                         {% if request.user.is_authenticated %}
-                        <a class="nav-link" href="/account">My Account ({{ request.user.username}})</a>
+                            <li class="nav-item dropdown " >
+                                <a class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Account ({{ request.user.username}})</a>
+                                <div class="dropdown-menu">
+                                    <a class="dropdown-item p-2 mb-1 bg-dark text-white" href="/account">My Account</a>
+                                    <div class="dropdown-divider"></div>
+                                    <a class="dropdown-item p-2 mb-1 bg-dark text-white" href="{% url 'account_logout' %}">Logout</a>
+                                 </div>
+                            </li>
```

Comment:
So I took a look at the bootstrap docs and it doesn't look like they include an easy way to customize the dropdown menu, so unfortunately you're probably going to have to write some css.

I did find this link though with some examples of customized bootstrap dropdowns that may be helpful in giving you an idea of what CSS modifications you will need. https://www.jquery-az.com/7-customized-bootstrap-dropdown-demos-in-menus-navbar-and-tabs/

Reply:
Done